src/views/Media/NewsletterSignup.vue

```diff
@@ -0,0 +1,233 @@
+                  <div class="field-body control">
+                    <input v-model.trim="stateProvince" class="input" type="text" placeholder="State/Province">
+                  </div>
+                </div> <!-- end of field -->
+                <div class="field is-horizontal">
+                  <label class="field-label cve-field-label">Country</label>
+                  <div class="field-body control">
+                    <input v-model.trim="country" class="input" type="text" placeholder="Country">
+                  </div>
+                </div> <!-- end of field -->
+                <div class="field is-horizontal">
+                  <label class="field-label cve-field-label"></label>
+                  <div class="field-body control">
+                    <input type="submit" value="Subscribe" class="button cve-button cve-base-color">
+                  </div>
+                </div> <!-- end of field -->
+              </form>
+            </div> <!-- end of container -->
+            <div v-if="submitted" class="notification is-success">
+              <h2 class="cve-top-margin">
+                <p id="successIcon" class="is-hidden">success</p>
+                <font-awesome-icon style="flex: 0 0 40px;" size="1x"  icon="check-circle" role="success"
+                 aria-labelledby="successIcon" aria-hidden="false" />
+                Success! Your subscription request has been submitted.
+              </h2>
+            </div>
+            <div class="mt-2">View our <router-link to=/Legal/PrivacyPolicy>Privacy Policy</router-link></div>
+          </div>
+          <h2 class="title">How to Unsubscribe</h2>
+          <p>
+          <a href="mailto:lms@mitre.org?Subject=signoff%20cve-announce-list">Click here</a> to send email to lms.mitre.org with
+            the subject: "signoff cve-announce-list". Alternatively, instructions on how to unsubscribe appear at the bottom of
+            each newsletter.
+          </p>
+        </main>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+
+export default {
+  name: 'NewsletterSignup',
+  components: {
+  },
+  data() {
+    return {
+      errors: [],
+      emailAddress: null,
+      mathAnswer: null,
+      firstName: '',
+      lastName: '',
+      jobFunction: 'unselected',
+      orgType: 'unselected',
+      organization: '',
+      cityTown: '',
+      stateProvince: '',
+      country: '',
+      submitted: false,
+    };
+  },
+  methods: {
+    checkForm(e) {
+      this.errors = [];
+      if (!this.emailAddress) {
+        this.errors.push('Email required.');
+      } else if (!this.validEmail(this.emailAddress)) {
+        this.errors.push('Valid email required.');
+      }
+
+      if (!this.mathAnswer) {
+        this.errors.push('Response to 9 + 4 required');
+      }
+
+      if (!this.errors.length) {
+        this.submitted = true;
+        return true;
+      }
+
+      e.preventDefault();
+      return false;
```

Comment:
Please remove 210-211, it's unnecessary.

Reply:
done